string new_map_name
---
string old_map_name